synfig-studio/src/gui/dialogs/dialog_pluginmanager.cpp

```diff
@@ -0,0 +1,161 @@
+/* === S Y N F I G ========================================================= */
+/*!	\file template.cpp
+**	\brief Template File
+**
+**	\legal
+**	Copyright (c) 2002-2005 Robert B. Quattlebaum Jr., Adrian Bentley
+**
+**	This file is part of Synfig.
+**
+**	Synfig is free software: you can redistribute it and/or modify
+**	it under the terms of the GNU General Public License as published by
+**	the Free Software Foundation, either version 2 of the License, or
+**	(at your option) any later version.
+**
+**	Synfig is distributed in the hope that it will be useful,
+**	but WITHOUT ANY WARRANTY; without even the implied warranty of
+**	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+**	GNU General Public License for more details.
+**
+**	You should have received a copy of the GNU General Public License
+**	along with Synfig.  If not, see <https://www.gnu.org/licenses/>.
+**	\endlegal
+*/
+/* ========================================================================= */
+
+/* === H E A D E R S ======================================================= */
+
+#ifdef USING_PCH
+#	include "pch.h"
+#else
+#ifdef HAVE_CONFIG_H
+#	include <config.h>
+#endif
+
+#include "dialog_pluginmanager.h"
+
+#include <gtkmm/label.h>
+#include <gtkmm/notebook.h>
+#include <gtkmm/listbox.h>
+#include <gtkmm/listboxrow.h>
+
+#include <gui/localization.h>
+#include <gui/app.h>
+
+#include <synfig/general.h>
+#include <synfig/os.h>
+
+#endif
+
+/* === U S I N G =========================================================== */
+
+using namespace synfig;
+using namespace studio;
+
+/* === M A C R O S ========================================================= */
+
+/* === G L O B A L S ======================================================= */
+
+/* === P R O C E D U R E S ================================================= */
+
+/* === M E T H O D S ======================================================= */
+
+Dialog_PluginManager::Dialog_PluginManager(Gtk::Window& parent):
+    Gtk::Dialog(_("Plugin Manager"), parent),
+    messageDialog(_("Are you sure you want to delete this plugin?"), false, Gtk::MESSAGE_QUESTION, Gtk::BUTTONS_OK_CANCEL, true),
+    pluginList(App::plugin_manager.plugins())
+{
+    // this->set_resizable(false);
+    messageDialog.set_transient_for(*this);
+    Gtk::Button *installPluginButton = manage(new Gtk::Button(_("Install Plugin")));
```

Comment:
Our (non-official?) code style now suggests (requires?):
`Gtk::Button* install_plugin_button`  instead of  `Gtk::Button *installPluginButton`.

1. No space before pointer mark `*`, but after
2. snake_case variable/method naming instead of camelCase

(There are other cases below)